### PR TITLE
[Feature] Icon component page

### PR DESCRIPTION
**Pull request by ketsappi**
Add icon-page to components

Tested by `yarn validate` and manually running in the browser.

Translations not yet pushed to Transifex; should be done after PR is done and translations have been checked thoroughly.

### CODE REVIEW

File: src/pages/components/icon.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+          <ComponentDescription
+            mainTitle={t('signLanguageIcons.title')}
+            description={t('signLanguageIcons.description')}
+            exampleFirst
+          >
+            <ComponentExample style={{ justifyContent: 'flex-start' }}>
+              <Icon icon="signLanguageContent" color="#00B6EC" />
+            </ComponentExample>
+          </ComponentDescription>
```
Comment (LJKaski) [inline src/pages/components/icon.tsx]: I think elsewhere the icons are presented without the borders in the examples. Could it work here as well?

Comment (ketsappi) [reply src/pages/components/icon.tsx]: Yeah, we can get rid of borders here.
Design in Abstract was having it so it was left as default.

File: locale/fi/icon.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "title": "Ikoni (Icon)",
+  "intro": "Ikonit ovat visuaalisia symboleja, jotka tukevat toimintojen ja aiheiden ilmaisemista. Ikonien avulla käyttäjän huomiota voidaan korostetusti kiinnittää haluttuun informaatioon tai toimintoon. Ikonien käyttäminen voi myös parantaaa laajojen sisältöjen hahmottamista ja ryhmittämistä. Suomi.fi Design Systemissä on sekä käyttöliittymäikoneita, että kuvitusikoneita.",
```
Comment (LJKaski) [inline locale/fi/icon.json]: Ennen viimeistä 'että' sanaa ei tule pilkkua.

File: locale/fi/icon.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Jos ikoni on informatiivinen tai toiminnallinen, kuvalle on luotava tekstivastine käyttämällä WAI-ARIA-määritettä 'aria-label' tai 'aria-labelledby'.",
+    "Jos ikoni on tarkoitettu vain visuaaliseksi tehosteeksi, käytetään sitä varten WAI-ARIA-määritettä role='presentation' (ei role='img'). Tällöin avustavat tekniikat ohittavat kuvan.",
+    "Kuvitusikonien käytössä tulee huomioida ikonille looginen käyttötarkoitus. Jos ikoni on ristiriidassa sisällön kanssa, tämä voi heikentää käytettävyyttä ja kognitiivista saavutettavuutta.",
+    "Ikonit ovat harvoin yksiselitteisiä kaikille käyttäjille. Suomifissa ikoni esitetään siksi yleensä tekstin kanssa, jolloin käyttäjän ymmärrys ei jää pelkän ikonin varaan. Poikkeuksena info-ikoni, jonka käyttö on yleisesti vakiintunut."
```
Comment (LJKaski) [inline locale/fi/icon.json]: Suomifi --> Suomi.fi Design System (todennäköisesti)

File: locale/fi/icon.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  "additionalInfoIcons.title": "Lisätietoikoni",
+  "additionalInfoIcons.description": "Lisätieto on asiaa, jonka käyttäjän täytyy lukea ja ymmärtää toimiakseen oikein. Lisätietoikoni on huomioarvoltaan suurempi kuin ohjeikoni. Tummalla pohjalla käytetään valkoista (whiteBase) ikonia. Lisätieto-ikonista on suomifi teemassa kaksi erillistä versiota, täytetty sekä ääriviiva versio. Täytettyä lisätieto-ikonia käytetään silloin kun sitä klikkaamalla käyttäjälle aukeaa lisätietoa. Ääriviiva versio lisätieto-ikonista käytetään visuaalisena tukena esimerkiksi modaaleissa.",
+  "illustrationIcons.title": "Kuvitusikonit",
+  "illustrationIcons.description": "Kuvitusikonit kuvaavat sisällössä esitettäviä elämäntilanteita ja aihealueita. Ikoneissa käytetyt värit ovat vaaleanharmaa (depthBase) ja oranssi (accentBase). Myös kuvitusikonit pyritään pitämään yksinkertaisina, jotta vältyttäisiin luomasta ylimääräistä kognitiivista rasitetta. Suomi.fi palveluissa jokaisella kuvitusikonilla on teema, joka antaa osviittaa käyttötarkoituksesta. Pyri valitsemaan kuvitusikoni joka mahdollisimman konkreettisesti vastaa käyttötarvetta, mitä vähemmän abstrakti sen parempi.",
```
Comment (LJKaski) [inline locale/fi/icon.json]: Viimeinen sivulause on aika turha ja toistaa käytännössä saman asian joka lauseessa jo kerrotaan. Pitää varmistaa muutos vielä designilta.

**Pull request merged**